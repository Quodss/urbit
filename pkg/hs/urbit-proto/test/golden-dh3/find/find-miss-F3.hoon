=/  a  p=1
q.a
