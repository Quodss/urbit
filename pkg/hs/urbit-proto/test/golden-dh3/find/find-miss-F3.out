PARSED

=/  a
  p=1
q.a

CODE

=+  a=p=1
q.a

ERROR

  trace:  play:
          lvl   0
          sut   *
          ken   0
          ----
          code  =+  a=p=1
                q.a


          play:
          lvl   0
          sut   *
          ken   0
          ----
          code  a=p=1

          play:
          lvl   0
          sut   *
          ken   0
          ----
          code  p=1

          nest:
          have  p=@ud
          need  $

          play:
          lvl   1
          sut   {* a=p=@ud}
          ken   [0 1]
          ----
          code  q.a

          find:
          lvl   1
          sut   {* a=p=@ud}
          ken   [0 1]
          ----
          wing  q.a


  find.q
  type  p=@ud

